services/user.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> None:
+    user = User.objects.create_user(

```

Comment:
Use `get_user_model()` instead

Reply:
Why?
get_user_model() takes id as a parameter. We do not take it as a parameter in create_user func.
"Create user.py, add such functions:
create_user, takes username, password, optional parameters email, first_name, last_name. The method should save user properly (password should be encrypted), use method .create_user() for this purpose."